packages/jest-haste-map/src/crawlers/__tests__/watchman.test.js

```diff
@@ -589,4 +589,89 @@ describe('watchman watch', () => {
     expect(calls[0][0]).toEqual(['list-capabilities']);
     expect(calls[2][0][2].fields).not.toContain('content.sha1hex');
   });
+
+  test('source control query', async () => {
```

Comment:
doesn't use `await`, but returns a promise

Reply:
Copy paste strikes again. I'll fix that up in the file generally.